.github/workflows/lava.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Adevinta
+
+name: Lava
+on: [push, pull_request]
+permissions:
+  contents: read
+jobs:
+  lava:
+    name: Lava
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Build vulnerability-db Docker image
+        run: docker build . -t vdb
```

Comment:
Do you think it worth to define a more generic step name and docker image name, for example:
```
      - name: Build the docker image
        run: docker build . -t imagetoscan
```

In this way, `.github/workflows/lava.yaml`, `lava.yaml` and `checktypes.json` files may be the same in all repositories.

Just as a suggestion.

Reply:
Applied. As we already discussed, this will probably change after defining the defaults and the distribution system. But, for now, I agree that the configs should be as similar as possible.